bigquery_etl/util/parallel_topological_sorter.py

```diff
@@ -74,16 +74,16 @@ def map(self, callback):
             ts.done(task)
             finalized_tasks_queue.task_done()
 
-            # add follow up tasks to be processed
-            if followup_queue:
-                while not followup_queue.empty():
-                    follow_up_item = followup_queue.get()
-                    ts.add(follow_up_item)
-                    self.visited[follow_up_item] = False
-
         task_queue.join()
         finalized_tasks_queue.join()
 
+        # followup items cannot be added to the topological sorter after prepare() is called
+        # processing them here sequentially
+        if followup_queue:
+            while not followup_queue.empty():
+                follow_up_item = followup_queue.get()
+                callback(follow_up_item, followup_queue)
+
```

Comment:
related to https://github.com/mozilla/bigquery-etl/pull/4091

I had to fix this so the downstream schema updates could run